src/twisted/internet/defer.py

```diff
@@ -1573,6 +1573,10 @@ class DeferredSemaphore(_ConcurrencyPrimitive):
     """
 
     def __init__(self, tokens):
+        """
+        @param tokens: initial value of C{tokens} and C{limit}
```

Comment:
Maybe the confusion is that source code markup like `C{limit}` often refers to a callable's argument, but there's no `limit` argument.